.eslintrc.json

```diff
@@ -1,28 +1,29 @@
 {
-  "parserOptions": {
-    "ecmaVersion": 5
-  },
+  "env": { "browser": true, "commonjs": true },
   "extends": "eslint:recommended",
-  "env": {
-    "commonjs": true
-  },
+  "parserOptions": { "ecmaVersion": 5 },
   "rules": {
-    "strict": [2, "global"],
-    "block-scoped-var": 2,
-    "consistent-return": 2,
-    "eqeqeq": [2, "smart"],
-    "guard-for-in": 2,
-    "no-caller": 2,
-    "no-extend-native": 2,
-    "no-loop-func": 2,
-    "no-new": 2,
-    "no-param-reassign": 2,
-    "no-return-assign": 2,
-    "no-unused-expressions": 2,
-    "no-use-before-define": 2,
-    "radix": [2, "always"],
-    "indent": [2, 2],
-    "quotes": [2, "double"],
-    "semi": [2, "always"]
```

Comment:
I'm not familar wit `eslintrc.json`'s configuration file, but I'm assuming `2` is a short-hand way of referring to `error`, correct?

Reply:
That's right.